ehrql/query_model/nodes.py

```diff
@@ -376,7 +376,7 @@ class FloorDivide(Series[int]):
 
     # Casting numeric types
     class CastToInt(Series[int]):
-        source: Series[Numeric]
+        source: Series[Numeric] | Series[bool]
```

Comment:
You could save **eight whole characters** by writing this as:
```py
Series[Numeric | bool]
```
_Eight_, Becky! What were you thinking?